Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h

```diff
@@ -74,12 +74,61 @@ DECLARE_SOA_COLUMN(TOFNSigmaHe, tofNSigmaHe, float);
 DECLARE_SOA_COLUMN(TOFNSigmaAl, tofNSigmaAl, float);
 } // namespace pidtof
 
+namespace pidtof_tiny
+{
+typedef int8_t binned_nsigma;
+constexpr int nbins = 1 << 8 * sizeof(binned_nsigma);
+constexpr float bin_max = 6.4;
+constexpr float bin_min = -6.4;
+constexpr float bin_width = (bin_max - bin_min) / nbins;
+// NSigma with reduced size 8 bit
+DECLARE_SOA_COLUMN(TOFNSigmaStoreEl, tofNSigmaStoreEl, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreMu, tofNSigmaStoreMu, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStorePi, tofNSigmaStorePi, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreKa, tofNSigmaStoreKa, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStorePr, tofNSigmaStorePr, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreDe, tofNSigmaStoreDe, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreTr, tofNSigmaStoreTr, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreHe, tofNSigmaStoreHe, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreAl, tofNSigmaStoreAl, binned_nsigma);
+// NSigma with reduced size in [bin_min, bin_max] bin size bin_width
+DECLARE_SOA_DYNAMIC_COLUMN(TOFNSigmaEl, tofNSigmaEl, [](float nsigma_binned) -> float {
+  return bin_min + bin_width * nsigma_binned;
+});
```

Comment:
Let's add a macro that does this
UNWRAP_COLUMN(TOFNSigmaEl, tofNSigmaEl, bin_min, bin_width)

Reply:
This is done!